domain/*.poderoculto.is-local.org

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "Dominio para site sobre poder oculto da mente e sedução",
+  "domain": "is-local.org",
+  "subdomain": "poderoculto",
```

Comment:
This subdomain doesn't match the file name. No valid reasoning was provided for why a wildcard domain is required for this project.